posthog-core/src/index.ts

```diff
@@ -616,23 +616,40 @@ export abstract class PostHogCoreStateless {
     )
   }
 
-  async shutdownAsync(): Promise<void> {
+  async shutdownAsync(shutdownTimeout?: number): Promise<void> {
```

Comment:
Just so its clear to the caller...
```suggestion
  async shutdownAsync(shutdownTimeoutMs?: number): Promise<void> {
```